amethyst_renderer/src/sprite.rs

```diff
@@ -19,17 +21,27 @@ pub type SpriteSheetHandle = Handle<SpriteSheet>;
 pub struct SpriteSheet {
     /// Index into `MaterialTextureSet` of the texture for this sprite sheet.
     pub texture_id: u64,
+    /// Pixel width of each sprite
+    pub sprite_w: f32,
```

Comment:
Sprites can have variable sizes in the same spritesheet

Reply:
* [ ] put it at the `Sprite` level